homework7/tests/test_hw1.py

```diff
@@ -0,0 +1,23 @@
+from homework7.hw1 import find_occurrences
+
+
+def test_find_str_in_dict_tree():
```

Comment:
add more test cases. what if tree is empty? what if there is no occurences? what if occurency is in key? etc.